.golangci.yml

```diff
@@ -172,6 +172,8 @@ linters-settings:
           - (github.com/golangci/golangci-lint/pkg/logutils.Log).Fatalf
     # enable all analyzers
     enable-all: true
+    disable:
+      - fieldalignment
```

Comment:
Because of some changes in `pfcp`, this check complains that about pointer sizes and alignment, which is an unnecessary optimization. It is disabled by default in golangci-lint. https://golangci-lint.run/usage/linters#govet